package.json

```diff
@@ -1,6 +1,14 @@
 {
     "name": "geo-web-core-contracts",
     "license": "MIT",
+    "main": "dist/src/index.js",
+    "types": "dist/src/index.d.ts",
+    "scripts": {
+      "build": "yarn build:sol && yarn build:ts && yarn build:declarations",
+      "build:ts": "tsc -p tsconfig.build.json",
+      "build:sol": "npx hardhat compile",
+      "build:declarations": "shx cp typechain/*.d.ts dist/typechain"
+    },
```

Comment:
added the shx dependency and copied the same sort of [build system seen in the `nouns-contracts` repo](https://github.com/nounsDAO/nouns-monorepo/blob/master/packages/nouns-contracts/package.json#L8-L14)